Geometry/MuonNumbering/src/MuonBaseNumber.cc

```diff
@@ -29,7 +29,7 @@ void MuonBaseNumber::addBase(const LevelBaseNumber& num) {
   }
   sortedBaseNumber.insert(cur, num);
 
-#ifdef LOCAL_DEBUG
+  //#ifdef LOCAL_DEBUG
```

Comment:
@slomeo @cvuosalo @silviodonato this update of ```MuonBaseNumber``` adds lots of (I guess) unwanted printouts, this changes the log file (but if ```LogInfo``` is silenced this is not noticed but in the summary), and as the output is different wrt the reference, the testDTGeometry unit test fails. @slomeo could you please fix this? Why removing the ```ifdef```?

Reply:
#28921